### PR TITLE
OSDOCS-11776: removes CCO content for Alicloud

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.16+

Issue:
[OSDOCS-11776](https://issues.redhat.com//browse/OSDOCS-11776)

Link to docs preview: 
* [Modes](https://80834--ocpdocs-pr.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator#about-cloud-credential-operator-modes_about-cloud-credential-operator)
* [Determining the Cloud Credential Operator mode](https://80834--ocpdocs-pr.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator#cco-determine-mode_about-cloud-credential-operator)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@jcpowermac PTAL

**Comment by jianping-shu** [comment]
lgtm

File: images/334_OpenShift_cluster_updating_and_CCO_workflows_0923_4.11_A_AliCloud_patch.png
Comment (skopacz1) [inline images/334_OpenShift_cluster_updating_and_CCO_workflows_0923_4.11_A_AliCloud_patch.png:1]: Just curious, if this is a new image, is the old one being deleted as well?

**Review by skopacz1 — APPROVED** [review]
One non-critical question, otherwise LGTM!

File: images/334_OpenShift_cluster_updating_and_CCO_workflows_0923_4.11_A_AliCloud_patch.png
Comment (jeana-redhat) [reply images/334_OpenShift_cluster_updating_and_CCO_workflows_0923_4.11_A_AliCloud_patch.png:1]: Probably a good policy in general, though I edited this one so kind of want to leave the other for accountability reasons until I can get an official replacement with a new number

**Comment by jeana-redhat** [comment]
Merging into main and CP into 4.17, will get change mgmt acks for 4.16 CP

**Pull request merged**